src/Game.tsx

```diff
@@ -62,32 +48,33 @@ export function Game({ names }: Props): JSX.Element {
   }, []);
 
   return (
-    <div>
+    <div className="page-wrapper">
       <Link to="/">
-        {''}
         <button className="button-close-game">Close game</button>
       </Link>
       <h1 className="header-text">Pokemons ping-pong</h1>
-      {names.length === 2 ? (
-        <div ref={setWrapperElem} className="game-wrapper">
-          <div ref={setFieldDivElem} className="field">
-            <div className="dotted-line"></div>
-            <Ball x={ballCoords.x} y={ballCoords.y} />
-          </div>
-          <PlayerCard
-            y={player1Coords.y}
-            x={player1Coords.x}
-            name={names[0]}
-          ></PlayerCard>
-          <PlayerCard
-            y={player2Coords.y}
-            x={player2Coords.x}
-            name={names[1]}
-          ></PlayerCard>
+
+      {pokemonNames.includes('') && <Redirect to="/" />}
+
+      <div ref={setWrapperElem} className="game-wrapper">
+        <div ref={setFieldDivElem} className="field">
+          <div className="dotted-line"></div>
+          <Ball
+            x={(ballCoords.x * (fieldSize * 0.95)) / 100}
+            y={(ballCoords.y * (fieldSize * 0.95)) / 100}
```

Comment:
`Ball` component doesn't know its real coordinates in the game, only UI coordinates. It brings a lot of limitations to it. For example, you can not start animation or change the ball's color when it touches an edge.

`Ball` component should accept game coordinates.

The same for `PlayerCard` component.